gemrb/GUIScripts/GUIOPT.py

```diff
@@ -646,6 +646,12 @@ def OpenQuitMsgWindow ():
 	Button.SetText (15589)
 	Button.SetEvent (IE_GUI_BUTTON_ON_PRESS, SaveGamePress)
 
+	EnteredGame = GemRB.GetSystemVariable(SV_ENTERED_GAME)
+	if EnteredGame:
+		Button.SetState(IE_GUI_BUTTON_ENABLED)
+	else:
+		Button.SetState(IE_GUI_BUTTON_DISABLED)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['IE_GUI_BUTTON_DISABLED' may be undefined, or defined from star imports: GUIDefines](https://app.codacy.com/app/gemrb/gemrb/pullRequest?prid=4591638)